tests/integration/workflows/conftest.py

```diff
@@ -93,6 +93,7 @@ def workflow_app(higgs_ontology):
             CELERY_RESULT_BACKEND='cache',
             CFG_BIBCATALOG_SYSTEM_RT_URL=RT_URL,
             DEBUG=True,
+            # FEATURE_FLAG_ENABLE_UPDATE_TO_LEGACY=True,
```

Comment:
Maybe should move, with the default value, to `config.py`, otherwise we lose track of these :smile: